lib/version_utils.pm

```diff
@@ -652,6 +653,16 @@ sub is_public_cloud {
     return get_var('PUBLIC_CLOUD');
 }
 
+=head2 is_openstack
```

Comment:
After reading this method, I wonder if we should name it `is_jeos_openstack`... wdyt?

Reply:
yeah, we can do that, OTOH I am not sure whether we can run an ISO installation in openstack. Thus I have kinda thought it is implicit

Reply:
Should be fine, but PC is also producing OpenStack images, e.g. `SLES15-SP4-CHOST-BYOS.aarch64-0.9.7-OpenStack-Build2.20.qcow2`

Reply:
Is that anything we might need to test in the future ? If so then it kinda makes sense to keep it or not ?

Reply:
> Is that anything we might need to test in the future ?
I really hope not :D 

 If so then it kinda makes sense to keep it or not ?
Merged :)